coding-guidelines/coding-guidelines.md

```diff
@@ -0,0 +1,243 @@
+# Coding Guidelines
+
+## Line length
+One Line in JavaScript should have a maximal length of 80 characters.
+
+## Indentation and Line-Breaks
+Rules:
+- Indentation is done with 2 spaces in most cases.
```

Comment:
This is clear from the `eslintrc` `indent` rule. No need to state the obvious imho.